content/en/docs/ambient/install/multicluster/_index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Install Multicluster
+description: Install an Istio mesh in ambient mode across multiple Kubernetes clusters.
+weight: 40
+aliases:
+    - /docs/ambient/kubernetes/multicluster-install/
+    - /docs/ambient/kubernetes/multicluster/
+    - /docs/ambient/kubernetes/install/multicluster/
+    - /docs/ambient/install/multicluster/gateways/
+    - /docs/ambient/install/multicluster/shared/
+keywords: [kubernetes,multicluster]
+simple_list: true
+content_above: true
+test: table-of-contents
+owner: istio/wg-environments-maintainers
+---
+
+{{< boilerplate alpha >}}
+
+Follow this guide to install an Istio {{< gloss "ambient" >}}ambient service mesh{{< /gloss >}}
+that spans multiple {{< gloss "cluster" >}}clusters{{< /gloss >}}.
+
+## Current Status and Limitations
+
+{{< warning >}}
+**Ambient multicluster is currently in alpha status** and has significant limitations.
+This feature is under active development and should not be used in production environments.
+{{< /warning >}}
+
+Before proceeding with ambient multicluster installation, it's critical to understand
+the current state and limitations of this feature:
+
+### Supported Configurations
+
+Currently, ambient multicluster **only supports**:
+- **Multi-network topologies** with multiple primary clusters
+- **Double HBONE encapsulation** for cross-cluster traffic
+- **Universal waypoint deployments** across all clusters with identical names
```

Comment:
waypoint deployment or configuration? You don't have to deploy waypoints in all clusters, but maybe we do want to require this to avoid confusion.

Reply:
Yes, but the services also have the same waypoints across clusters

Reply:
I guess we talk about it later